docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -37,13 +37,48 @@ Custom Search Attributes of the `Text` type cannot be used in **ORDER BY** claus
 
 ### Partial string match
 
-The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
+#### Text
 
-<!-- note: advanced vis features will be supported in SQL upon the release of v1.20.-->
+Search Attributes of type `Text` are broken up into words that match with the `=` operator.
 
-For example, if you have a custom Search Attribute named `Description` of `Text` type with the value of "The quick brown fox jumps over the lazy dog", searching for `Description='quick'` or `Description='fox'` will successfully return the Workflow.
-However, partial word searches such as `Description='qui'` or `Description='laz'` will not return the Workflow.
-This is because [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
+For example, if you have a custom `Text` Search Attribute named `Description` with either of the following values:
```

Comment:
```suggestion
For example, if you have a custom `Text` Search Attribute named `Description` with either of the following values—
```
This and the related following edits are an informal approach to the grammar. Another approach is to make each intro line a complete sentence—entirely valid but does add words.